apps/frontend/src/hooks/aave/utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { utils } from 'ethers';
+
+import { Decimal } from '@sovryn/utils';
+
+import { calculateUtilizationRate, isBetweenZeroAndOne } from './utils';
+
+describe('hooks/aave/utils', () => {
```

Comment:
liked it 👍 .
I'm just wondering if this is the place for testing files

Reply:
Agree, I think is a good intention, but we didn't do tests for all the other calculations, and they neither have tests for theirs, so just for one I wouldn't open the door for requests

Reply:
Agree to remove the test file ? @matzapata @juandahl